packages/office-ui-fabric-react/src/Shimmered.ts

```diff
@@ -0,0 +1 @@
+export * from './components/DetailsList/ShimmeredDetailsList';
```

Comment:
Should the top level bundle be called ShimmeredDetailsList? (unless we anticipate adding more shimmered components here over time?)

Reply:
I thought the same thing

Reply:
ping @ThomasMichon